### ISSUE TITLE
Last line not correctly updated in some cases

### ISSUE THREAD

**Issue by prso** [prso/hexcurse#2]
To reproduce the problem:

Open any file and go to the end (END key):
![hexc1x](https://user-images.githubusercontent.com/19615588/179430959-68a97ae4-85a8-4d4a-ac9f-fb8f3be40cb7.png)

do PGUP:
![hexc2x](https://user-images.githubusercontent.com/19615588/179431019-daf02886-86ff-47a5-89ec-8b18b75eece3.png)

do UP and PGDOWN, now you are again at the end of the file but you can see that the last 0x2A values (highlighted in a red rectangle) shouldn't be there as they are the values of the previous page, the last byte of the file is 0x0A:
![hexc3xx](https://user-images.githubusercontent.com/19615588/179431464-b557b1bf-cfd9-44fd-b726-929b00652d1d.png)

This bug doesn't affect the content of the file, only the display of the last line.

The file used for the example is file.c from the sources.

**Comment by prso** [comment]
I have discovered this is due to (n)curses function wclrtoeol() not working in the case described but unfortunately I don't know the reason.

I have applied a workaround: when wclrtoeol() fails the empty positions of the last line are cleared one by one. Commit: https://github.com/prso/hexcurse/commit/a6cef7ba55a371e46df5b8f11c4f995dc36bc2bb

I will keep this issue open just in case somebody has a better solution that would be welcome.
